src/index.html

```diff
@@ -2,13 +2,54 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+              maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a class="logo-link" href="#home">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo Moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li><a class="nav__link is-active" href="#">Apple</a></li>
```

Comment:
```suggestion
          <li>
            <a class="nav__link is-active" href="#">Apple</a>
          </li>
```
preferred formatting for nested tags (FYI)